internal/manifest/filemutex/cache_test.go

```diff
@@ -0,0 +1,51 @@
+package filemutex_test
+
+import (
+	"sync"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/kyma-project/lifecycle-manager/internal/manifest/filemutex"
+)
+
+func TestNewMutexCache_WhenCalled_NotNil(t *testing.T) {
```

Comment:
The other PR has an explicit test for the constructor with a cache object being passed in. I consider it tested implicitly with all the tests below. But it probably wouldn't harm to have it explicitly included here as well.

Reply:
True, added.